rfps/privacy-enhancement-polkadot-extension.md

```diff
@@ -0,0 +1,34 @@
+# Privacy Enhancement for Polkadot Extension
+
+* **Status:** Open
+* **Proposer:** jonasW3F
+* **Your Project(s):** [optional]: N/A
+* **Projects you think this work could be useful for** [optional]: https://github.com/polkadot-js/extension
+* **Teams/People that could deliver the RFP** [optional]: N/A
+
+## Project Description :page_facing_up: 
+
+An increasing number of websites require access to the Polkadot extension with a growing ecosystem. By allowing access to the extension, websites (naturally) can see the addresses that are contained in the extension. This imposes a big risk to privacy, because malicious actors could create lists about which addresses belong to one entity and, in the worst case, even link it to real-world identities (if at least one address is linked to an on-chain identity). A feature to prevent this is currently the "hide" button on single addresses, which prevent websites from seeing that address. However, it is currently cumbersome to handle that feature. The idea of this RFP is to extend on that feature and include a few QOL functionalities to make it easier for users to protect their privacy.
+
+## Deliverables :nut_and_bolt:
+
+As outlined [here](https://github.com/polkadot-js/extension/issues/893), the deliverable should include five features to the extension and a successful completion of this RFP includes a merge of a PR to the main polkadot-js/extension repo. **It is of great importance to generate clean code that follows best-practices**. 
+
+* **Total Estimated Duration:** 1 month
+* **Full-time equivalent (FTE):**  Amount of time (in days) required for a single person to complete this project ([see](https://en.wikipedia.org/wiki/Full-time_equivalent)) 
+* **Total Costs:** $10'000. 
+
+### Milestone 1
+ 
+* **Estimated Duration:** 1 month
+* **FTE:**  Amount of time (in days) required for a single person to complete this milestone
+* **Costs:** $10'000
```

Comment:
```suggestion
* **Costs:** DAI 10,000
```

Reply:
I'd actually leave this, pricing should be in USD while payment can be in any currency